app/Models/CategoryDAO.php

```diff
@@ -45,6 +45,12 @@ protected function addColumn($name) {
 				} else {
 					$this->pdo->exec('DROP INDEX IF EXISTS feed_keep_history_index');	//SQLite at least drop index
 				}
+
+				$stm = $this->pdo->prepare('UPDATE `_category` SET name = :name WHERE id = :id');
+				$stm->bindValue(':id', self::DEFAULTCATEGORYID, PDO::PARAM_INT);
+				$stm->bindValue(':name', 'Uncategorized');
+				$stm->execute();
```

Comment:
In `addColumn`? :thinking:

Reply:
Yes, this could indeed be refactored or renamed, but it is only a trigger based on a lacking column, with a set of related updates

Reply:
well, looking at the rest of the code, it's not that surprising (`DROP` :p) but still a bit odd :upside_down_face:

Reply:
Indeed...